atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/TransactionManager.java

```diff
@@ -433,6 +434,17 @@ default LockWatchManager getLockWatchManager() {
     @Timed
     TransactionAndImmutableTsLock setupRunTaskWithConditionThrowOnConflict(PreCommitCondition condition);
```

Comment:
We need to remove this method, since it is only supposed to be used by internal ski product.